sched/queue/queue.go

```diff
@@ -12,6 +12,11 @@ type Queue interface {
 	// Enqueue enqueues a job and assigns it an ID. Errors may be
 	// an *InvalidJobRequest or a *CanNotScheduleNow
 	Enqueue(job sched.Job) (string, error)
+
+	Chan() chan WorkItem
+
+	Status(jobId string) (sched.JobStatus, error)
```

Comment:
Is this something we plan to implement on the Queue?  
Getting status from a durable queue is going to be expensive, requires reading the whole queue every time.